### ISSUE TITLE
extend notification for mails without subject

### ISSUE THREAD

**Issue by tooomm** [mkiol/GNotifier#154]
win 10 64bit
gnotifier 1.9.8

if you use this kind of settings:
![settings](https://cloud.githubusercontent.com/assets/9874850/22399413/4adc55a0-e59c-11e6-9307-619a33b1b250.png)
With `%s` for `subject` there is the problem that emails without subject at all just show the "sender name" in the notification, noting else. So you can't tell for sure if something is going wrong, you changed the settings or why it's not displayed. It's intended behavior in the first place, but could be improved.
I think it would be better to tell the user that the mail actually had no subject to stay uniform with a two raw layout in the notification, and to be clear in the message communicated.

like:
```
"Name of Sender"
"no/empty/missing subject", "-", "/", or something like "Email doesn't contain a subject" (maybe even in itallic to separate it from a real subject with the same wording)
```

I run into an empty-subject message today and wondered if an auto-plugin-update changed something at first. :)

**Comment by mkiol** [comment]
It is good idea. Thanks for suggestion.

**Comment by mkiol** [comment]
Resolved in the >=1.10.0 versions.
